Create the 21.2.3+ branch (#1)
DevExpress-Examples/asp-net-web-forms-popup-show-popup-window: Readme.md

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E55)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Popup Control for ASP.NET Web Forms - How to show the Popup Control
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e55/)**
<!-- run online end -->
This example demonstrates how to show a pop-up window.

## Overview

> Handle a client-side or server-side event and call one of the following methods to show the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl)'s pop-up window:

Reviewer: `client-side or server-side event` Any?

Author: I've rewrite the Readme's text. Could you please review it again?

Displays the default or specific pop-up window at a popup element with the specified index. Use the [PopupHorizontalAlign](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.PopupHorizontalAlign), [PopupHorizontalOffset](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.PopupHorizontalOffset), [PopupVerticalAlign](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.PopupVerticalAlign), and [PopupVerticalOffset](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.PopupVerticalAlign) properties to specify the window's position relative to the element.

* **[ShowAtElement](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.ShowAtElement(htmlElement))** or **[ShowWindowAtElement](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControl.ShowWindowAtElement(window-htmlElement))**  
Displays the default or specific pop-up window over the specified HTML element.

* **[ShowAtElementByID](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControlBase.ShowAtElementByID(id))** or **[ShowWindowAtElementByID](https://docs.devexpress.com/AspNet/js-ASPxClientPopupControl.ShowWindowAtElementByID(window-id))**  

> Displays the default or specific pop-up window over an HTML element with the specified identifier.

Reviewer: `or specific` Which specific?

[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E55)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->
# Popup Control for ASP.NET Web Forms - How to show a pop-up window
<!-- run online -->
**[[Run Online]](https://codecentral.devexpress.com/e55/)**
<!-- run online end -->
This example demonstrates various ways to display a default pop-up window.

## Overview

> The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) creates a default pop-up window when the control's [Windows](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.Windows) collection is empty. You can change the default pop-up window's visibility on the server and client sides.

Reviewer: ```suggestion
The [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) creates a default pop-up window when the control's [Windows](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl.Windows) collection is empty. You can change the default pop-up window's visibility on the server or client.
```
